MODULE.bazel

```diff
@@ -7,9 +7,8 @@ bazel_dep(name = "rules_player")
 
 archive_override(
   module_name = "rules_player",
-  strip_prefix = "rules_player-1.0.1",
-  urls = ["https://github.com/player-ui/rules_player/archive/refs/tags/v1.0.1.tar.gz"],
-  integrity = "sha256-zIbxx1Bh/X4E8OwbPJz1nY9mOd2uAgmTZyf1Hs2mq+U="
+  strip_prefix = "rules_player-1.1.0",
+  urls = ["https://github.com/player-ui/rules_player/archive/refs/tags/v1.1.0.tar.gz"],
```

Comment:
Any reason we dropped the integrity check?

Reply:
i can pull in the integrity for 1.1.0!